Marlin/src/inc/Conditionals_post.h

```diff
@@ -300,7 +300,7 @@
 /**
  * Override here because this is set in Configuration_adv.h
  */
-#if HAS_LCD_MENU && DISABLED(ELB_FULL_GRAPHIC_CONTROLLER)
+#if HAS_LCD_MENU && DISABLED(ELB_FULL_GRAPHIC_CONTROLLER) && !(defined(ARDUINO_GRAND_CENTRAL_M4) && SD_CONNECTION_IS(ONBOARD))
```

Comment:
@GMagician — Should this instead be …
```
#if (HAS_LCD_MENU && DISABLED(ELB_FULL_GRAPHIC_CONTROLLER)) \
  || (!SD_CONNECTION_IS(ONBOARD) || !defined(ARDUINO_GRAND_CENTRAL_M4))
```
The previous condition was to deal with the SD pin on `ELB_FULL_GRAPHIC_CONTROLLER` not being inverted, since most LCDs do want this to be inverted. So if that's not also the case with every GCM4 non-onboard SD card reader, then this condition should perhaps just be removed.